src/components/NurseryWithdrawals/PlantingProgressMap.tsx

```diff
@@ -73,48 +86,74 @@ export default function PlantingProgressMap({ plantingSiteId }: PlantingProgress
       if (updateStatus.status === 'error') {
         snackbar.toastError(strings.GENERIC_ERROR);
       } else if (updateStatus.status === 'success') {
-        // refresh planting site data to get new completed state for subzone
+        // refresh planting site and observations data
         dispatch(requestPlantingSites(org.selectedOrganization.id));
+        dispatch(requestObservationsResults(org.selectedOrganization.id));
```

Comment:
There is a function _reloadTrackingAndObservations_ in the parent (Planting progress tab content) that refreshes this. Maybe we can pass it to the map as we are passing it to the list? It also refreshes plantingSites

Reply:
Oh, I'll take a look at using that 👍

Reply:
I'm not sure why but it caused the map to glitch like crazy. I'm going to leave as is for now, and we can possibly follow up.

Reply:
I will take a look at this today.